cmd/podman/run.go

```diff
@@ -64,8 +65,12 @@ func runCmd(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
-
-	createConfig, err := parseCreateOpts(c, runtime, newImage.Names()[0], data)
+	if len(newImage.Names()) < 1 {
+		imageName = ""
```

Comment:
Would it make sense to set imageName to the image ID in this case?

Reply:
you tell me

Reply:
It looks like we really only use this for Inspect output, so this probably isn't necessary

Reply:
They default to imageID from what I could see.  I didn't find evidence of Docker actually storing the shortImageId anywhere, only generating it on the fly, but may have missed it in my search this morn.  I'd personally vote for the full imageID if no name was found.

Reply:
updated to use long id